### PR TITLE
feat: add app dir example

### PR DESCRIPTION
**Pull request by 2wce**

### CODE REVIEW

File: examples/next-13-app-router-graphql/app/api/draft/route.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { draftMode } from 'next/headers';
+import { redirect } from 'next/navigation';
+
+export async function GET(request: Request) {
+  // Parse query string parameters
+  const { searchParams } = new URL(request.url);
+  const secret = searchParams.get('secret');
+  const slug = searchParams.get('slug');
+
+  console.log({ secret, slug, env: process.env.CONTENTFUL_PREVIEW_SECRET });
```
Comment (2wce) [inline examples/next-13-app-router-graphql/app/api/draft/route.ts:12]: will clean up remove this

File: examples/next-13-app-router-graphql/app/providers.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+'use client';
```
Comment (2wce) [inline examples/next-13-app-router-graphql/app/providers.tsx:1]: @z0al this is the part i was talking about, without this `use client` we are getting an error even though the provider has it's own directive

File: examples/next-13-app-router-graphql/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```
Comment (YvesRijckaert) [inline examples/next-13-app-router-graphql/README.md:1]: Can we use the same docs as the other examples?

**Comment by YvesRijckaert** [comment]
It would be nice to also mention this in the root README file, if they are using the app dir that they can check out this example on how to integrate it

File: examples/next-13-app-router-graphql/app/api/draft/route.ts [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+import { redirect } from 'next/navigation';
+
+export async function GET(request: Request) {
+  // Parse query string parameters
+  const { searchParams } = new URL(request.url);
+  const secret = searchParams.get('secret');
+  const slug = searchParams.get('slug');
+  const { isEnabled } = draftMode();
+
+  console.log({ secret, slug, env: process.env.CONTENTFUL_PREVIEW_SECRET, isEnabled });
```
Comment (YvesRijckaert) [inline examples/next-13-app-router-graphql/app/api/draft/route.ts:13]: let's remove the console log

**Review by YvesRijckaert — APPROVED** [review]

**Pull request merged**